pyfr/plugins/fluidforce.py

```diff
@@ -39,6 +39,14 @@ def __init__(self, intg, cfgsect, suffix):
         # Boundary to integrate over
         bc = f'bcon_{suffix}_p{intg.rallocs.prank}'
 
+        # Moments
+        mcomp = 3 if self.ndims == 3 else 1
+        self._mcomp = mcomp if self.cfg.hasopt(cfgsect, 'morigin') else 0
+        if self._mcomp:
+            morigin = np.array(self.cfg.getliteral(cfgsect, 'morigin'))
+            if morigin.shape[1] != self.ndims:
```

Comment:
morigin should be a 1D array (so specified as (x, y, z) or [x, y, z]).

Reply:
True. This is fixed now.